pkg/client.go

```diff
@@ -790,6 +791,34 @@ func (o *Client) InviteUserToOrg(params models.InviteUserToOrg) (bool, error) {
 	return true, nil
 }
 
+// ResendEmailConfirmation will resend the email confirmation email to a user.
+func (o *Client) ResendEmailConfirmation(userID uuid.UUID) (bool, error) {
+	urlPostfix := "resend_email_confirmation"
+
+	type ResendEmailConfirmationParams struct {
+		UserID uuid.UUID `json:"user_id"`
+	}
+
+	bodyParams := ResendEmailConfirmationParams{
+		UserID: userID,
+	}
+	bodyJSON, err := json.Marshal(bodyParams)
+	if err != nil {
+		return false, fmt.Errorf("Error on marshalling body params: %w", err)
+	}
+
+	queryResponse, err := o.queryHelper.Post(o.integrationAPIKey, urlPostfix, nil, bodyJSON)
+	if err != nil {
+		return false, fmt.Errorf("Error on resending email confirmation to user: %w", err)
+	}
+
+	if err := o.returnErrorMessageIfNotOk(queryResponse); err != nil {
+		return false, fmt.Errorf("Error on resending email confirmation to user: %w", err)
```

Comment:
isn't this unreachable?

Reply:
We do it in our endpoints everywhere else, so it's a safety precaution